webapp-django/crashstats/settings/base.py

```diff
@@ -346,6 +347,12 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Rate limit for when using the Web API for anonymous hits
 API_RATE_LIMIT = '10/m'
 
+# Rate limit when using the supersearch web interface
+RATELIMIT_SUPERSEARCH = '5/m'
```

Comment:
Perhaps this is out of scope, but would it be possible to set different limits based on whether the user is logged in or not?